pkgs/data/fonts/ir-standard-fonts/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "ir-standard-fonts";
```

Comment:
The package name should follow the [package naming guidelines](https://nixos.org/nixpkgs/manual/#sec-package-naming). I.e., please change this line to
```
name = "ir-standard-fonts-unstable-2017-01-21";
```

Reply:
I'll add a version and change name to pname like this:
https://github.com/NixOS/nixpkgs/pull/58128/files
Is it ok?